hs/CssGen.hs

```diff
@@ -650,4 +652,79 @@ fceCountCSS = "#FCECountDiv" ? do
     "vertical-align" -: "middle"
     fontSize (em 1.35)
     alignCenter
-
+   
+-- Post Styles
+postStyles = do
+    body ?
+        do backgroundColor $ "#EBE8E4"
```

Comment:
There are `$` here that are redundant; also, please try to use existing colours we've defined in this file, or just add your own. This will be helpful when looking at colours later on.